style.css

```diff
@@ -0,0 +1,99 @@
+body {
+  background-color: beige;
+}
+
+.main {
+  position: absolute;
+  border: 1px dotted blue;
+  width: 375px;
+  height: 812px;
+  background-image: url("image/Ellipse11.png"), url("./image/Rectangle7.png"), url("./image/Ellipse3.png"), url("./image/Ellipse12.png"), url("./image/Ellipse4.png"), url("./image/header-background-mobile.png");
```

Comment:
- [ ] I think you are using the correct images for this project. However, I believe you mistyped the names of the images. That is why the background images are missing / not rendering.

Your Design |
:---------------:|
![a1](https://user-images.githubusercontent.com/39523312/176891633-2cd49ed2-c5e9-4d5a-ab98-a01e7f85adae.png)


For example, there should be a space between the word `Ellipse` and the number `11`.
Because that's the way it is inside the image directory.

In that case, the `url("image/Ellipse11.png")`  should be `url("image/Ellipse 11.png")`  Please take note of the space.

Please correct the file names in order to get your images to render on your portfolio correct as in your chosen Figma design.

Reply:
Problem identified and the correct file is updated.